environments/stg/stg.tfvars

```diff
@@ -207,6 +207,13 @@ frontends = [
     backend_domain   = ["firewall-prod-int-palo-aat.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-aat-platform-hmcts-net"
   },
+  {
+    name             = "sscs-tya-redirect"
+    mode             = "Detection"
+    custom_domain    = "sscs-cor.aat.platform.hmcts.net/validate-surname/:tya/trackyourappeal"
```

Comment:
Is this correct? I think this needs to be just a custom domain name without "/"

Reply:
its for redirecting http://track-appeal.aat.platform.hmcts.net/* 
to 
http://sscs-cor.aat.platform.hmcts.net/validate-surname/:tya/trackyourappeal
should I still change it to sscs-cor.aat.platform.hmcts.net

Reply:
not sure if this will work. As this is supposed to be a domain and not url.